readme.md

```diff
@@ -2,45 +2,66 @@
+> plugins in the Babel organization that may be useful to you.
 
 If you want to contribute, please read the [contribution guidelines](contributing.md).
 
-## Table of Contents
-
-- [Parsers](#parsers)
-- [Plugins](#plugins)
- - [Frameworks and Libraries](#frameworks-and-libraries)
- - [Testing](#testing)
-- [Presets](#presets)
-
 ## Parsers
 
  - [babel-eslint](https://github.com/babel/babel-eslint) - ESLint using Babel as the parser.
+   - Note: ESLint now lints most ES6+ syntax. This parser is only necessary if you are using Flow types or other experimental features.
 
 ## Plugins
 
-### Frameworks and Libraries
+### General Plugins
 
- - [React Optimize](https://github.com/thejameskyle/babel-react-optimize) - A Babel preset and plugins for optimizing React code.
- - [React Transform](https://github.com/gaearon/babel-plugin-react-transform) - Babel plugin to instrument React components with custom transforms
- - [c-3po](https://c-3po.js.org) - javascript localization tool based on es6 template literals which is implemented as a babel plugin.
- - [module-resolver](https://www.npmjs.com/package/babel-plugin-module-resolver) - Custom module resolver
- - [lodash](https://www.npmjs.com/package/babel-plugin-lodash) - Cherry-picks Lodash modules so you don’t have to.
- - [transform-react-remove-prop-types](https://www.npmjs.com/package/babel-plugin-transform-react-remove-prop-types) - Remove unnecessary React propTypes from the production build.
- - [react-require](https://www.npmjs.com/package/babel-plugin-react-require) - adds React import declaration if file contains JSX tags.
  - [emojification](https://www.npmjs.com/package/babel-plugin-emojification) - Turn all binding identifiers to unicode escaped emoji :smirk_cat:
- - [idx](https://github.com/facebookincubator/idx) - library + babel plugin for a existential function
- - [react-intl](https://www.npmjs.com/package/babel-plugin-react-intl) - Extracts string messages for translation from modules that use [React Intl](https://github.com/yahoo/react-intl).
+ - [external-helpers](https://www.npmjs.com/package/babel-plugin-external-helpers) - Moves helper functions to a single imported module.
+ - [fast-async](https://github.com/MatAtBread/fast-async) - Uses nodent to compile async/await to fast Promise output.
+ - [feature-flags](https://github.com/ember-cli/babel-plugin-feature-flags) - Helper for managing application feature flags.
+ - [idx](https://github.com/facebookincubator/idx) - library + babel plugin for a existential function.
+ - [transform-builtin-extend](https://github.com/loganfsmyth/babel-plugin-transform-builtin-extend) - Enable extending builtin types like `Error` and `Array`, which require special treatment and require static analysis to detect.
+ - [transform-decorators-legacy](https://github.com/loganfsmyth/babel-plugin-transform-decorators-legacy) - Restores Babel 5 decorator output. Caution: Decorators are very likely to change.
```

Comment:
Yeah, I agree, I think it should be removed. People shouldn't be using this plugin.